utils.py

```diff
@@ -118,7 +120,7 @@ def write_output(config, results_list):
         logging.warning("Output path %s does not exist, creating it!", path)
         path.mkdir(parents=True, exist_ok=True)
 
-    concurrency, duration, _ = parse_config(config)
+    config["load_options"]["concurrency"] = concurrency
```

Comment:
This seems unnecessary as the value in concurrency is already overwritten in the calling method (load_test.py@146).

Reply:
Good catch, thanks!